extension/intents/forms/formsContentScript.js

```diff
@@ -176,6 +176,14 @@ this.dictationContentScript = (function() {
     return focusDirection(1);
   });
 
+  communicate.register("formSubmit", async message => {
+    onInputKeyPress = event(e) => {
+      if (event.key === "Enter") {
+        e.submitTextInput();
+      }
+    };
```

Comment:
I don't think this will do anything? Or at least I'm unclear what it will do, it seems to be creating a global function, but I don't think anything will ever call this function.

What should happen here is that it actually submits the form. Probably more like `document.activeElement.form.submit()` – though there might be some caveats depending on what element is active.